src/client.ts

```diff
@@ -115,6 +123,27 @@ export class FarosClient {
     }
   }
 
+  async addCanonicalModels(
+    graph: string,
+    models?: string,
+    schema = 'canonical'
+  ): Promise<void> {
+    if (this.graphVersion !== GraphVersion.V1) {
+      throw new VError(
+        `Adding models is not supported for ${this.graphVersion} graphs`
+      );
+    }
+    try {
+      await this.api.post(
+        `/graphs/${graph}/models?schema=${schema}`,
+        models ?? '',
```

Comment:
would empty string work here?

Reply:
That's what was passed [here](https://github.com/faros-ai/canonical-reports/blob/5a4bbdd83be2098e50fa8c2194597f3f07729d77/src/clients/faros.ts#L78)

Although after seeing the [endpoint implementation](https://github.com/faros-ai/clio/blob/40aed9799a7739cf3b329ed1536085d7464583a6/server/src/manager/index.ts#L188-L200) I think we should have `addCanonicalModels` which passes `schema = 'canonical'` and another `addModels` with required `models: string`

Reply:
I made these changes ^